modules/cnf-installing-sriov-fec-operator.adoc

```diff
@@ -0,0 +1,170 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="installing-the-sriov-fec-operator_{context}"]
```

Comment:
I doubt that the civilization crumbles if the ID doesn't *exactly* match the module file name, but it's written down.  See #2:

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#module-file-metadata